### PR TITLE
Cleanup Kibana Parameter Source

### PR DESCRIPTION
**Pull request by danielmitterdorfer**
With this commit we remove some dead code from the Kibana parameter source. We
also turn `dashboard` into a mandatory parameter and add tests.

### CODE REVIEW

File: tests/parameter_sources/elasticlogs_bulk_source_test.py [hunk 1]
```diff
@@ -15,11 +16,6 @@ def generate_event(self):
         return self.doc, self.index, self.type
 
 
-class StaticTrack:
```
Comment (dliappis) [inline tests/parameter_sources/elasticlogs_bulk_source_test.py]: Was this removed on purpose?

It is used in tests [here](https://github.com/elastic/rally-eventdata-track/pull/53/files#diff-77001cf4fc758355046aa067185cf830R24), [here](https://github.com/danielmitterdorfer/rally-eventdata-track/blob/321edf8aaff0a209925c73fa10b1feb1f4661c8f/tests/parameter_sources/elasticlogs_kibana_source_test.py#L31) and here: https://github.com/danielmitterdorfer/rally-eventdata-track/blob/321edf8aaff0a209925c73fa10b1feb1f4661c8f/tests/parameter_sources/elasticlogs_bulk_source_test.py#L2

Comment (danielmitterdorfer) [reply tests/parameter_sources/elasticlogs_bulk_source_test.py]: It was not removed but moved to `__init__.py`. For some reason the import worked when running in the IDE but I also noticed the tests where failing on the command line and fixed it in https://github.com/elastic/rally-eventdata-track/pull/53/commits/ccfd495cb1f78d5a6879f1b64ebb4ba086143954. Thanks for catching.

File: eventdata/parameter_sources/elasticlogs_kibana_source.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ class ElasticlogsKibanaSource:
     Simulates a set of sample Kibana dashboards for the elasticlogs data set.
 
     It expects the parameter hash to contain the following keys:
-        "dashboard"             -   String indicating which dashboard to simulate. Options are 'traffic', 'content_issues' and 'discover'. Defaults to 'traffic'.
+        "dashboard"             -   String indicating which dashboard to simulate. Options are 'traffic', 'content_issues' and 'discover'.
```
[the remark this answers is not in the document]
Comment (danielmitterdorfer) [reply eventdata/parameter_sources/elasticlogs_kibana_source.py]: The values are already documented in the README but I've corrected it now to mention it is three not two dashboards in https://github.com/elastic/rally-eventdata-track/pull/53/commits/6d494bc103f87688c8fe006387904600dd4987ea.

[the remark this answers is not in the document]
Comment (dliappis) [reply eventdata/parameter_sources/elasticlogs_kibana_source.py]: Thanks. What I am not seeing in the README is the _name_ of the parameter; ok it's mandatory so things won't progress much without it, but I think we should document that the mandatory parameter _name_ is **dashboard** and the eligible values are the following three.

[the remark this answers is not in the document]
Comment (danielmitterdorfer) [reply eventdata/parameter_sources/elasticlogs_kibana_source.py]: Addressed in https://github.com/elastic/rally-eventdata-track/pull/53/commits/7997679fef72cad1d6df4b5ea1b833b8347f8cbf.

**Pull request merged**